scenarios4/17_Any_Means_Necessary.cfg

```diff
@@ -85,8 +85,12 @@
         canrecruit=yes
         side=1
         controller=human
+#ifdef HARD
+        recruit=Elvish Archer,Elvish Fighter,Elvish Scout,Elvish Shaman,Duelist,Horseman,Spearman,Fencer,Heavy Infantryman,Elvish Lord
+#else
         recruit=Elvish Archer,Elvish Fighter,Elvish Scout,Elvish Shaman,Elvish Marksman,Elvish Ranger,Elvish Hero,Elvish Captain, Elvish Rider,Elvish Druid,Elvish Sorceress,Lancer,Longbowman,Duelist,Javelineer,Pikeman,Swordsman,Dragoon,Shock Trooper,Horseman,Knight,Cavalryman,Spearman,Fencer,Heavy Infantryman,Elvish Lord
-        gold=1500
+#endif
+        {GOLD 1500 1500 700}
```

Comment:
1200 gold on normal, please.

Reply:
Done.  But again, changing normal only because hard changed?

Reply:
Yes. There should be some kind of progression, if a scenario is too easy on hard, it's probably too easy also on normal.

Reply:
Maybe so.  Or there wasn't much difference between normal and hard to begin with.